### PR TITLE
[SMALLFIX] Update the s3 integration doc.

### PR DESCRIPTION
**Pull request by calvinjia**
Removes references to s3n.

### CODE REVIEW

File: docs/en/Configuring-Alluxio-with-S3.md [hunk 1]
```diff
@@ -75,42 +62,35 @@ Alternatively, these configuration settings can be set in the `conf/alluxio-env.
...
+### DNS-Buckets
 
-The underlying S3 library JetS3t can incorporate bucket names that are DNS-compatible into the host
-name of its requests. You can optionally configure this behavior in the `ALLUXIO_JAVA_OPTS` section
-of the `conf/alluxio-site.properties` file by adding:
+By default, a request directed at the bucket named "mybucket" will be sent to the host name
+"mybucket.s3.amazonaws.com". You can enable DNS-Buckets to use path style data access, for example:
+"http://s3.amazonaws.com/mybucket" by setting the following configuration:
 
 {% include Configuring-Alluxio-with-S3/jets3t.md %}
```
Comment (madanadit) [inline docs/en/Configuring-Alluxio-with-S3.md]: rename the file or inline?

Comment (calvinjia) [reply docs/en/Configuring-Alluxio-with-S3.md]: Good call, I've in-lined the snippets.

File: docs/en/Configuring-Alluxio-with-S3.md [hunk 2]
```diff
@@ -75,42 +62,35 @@ Alternatively, these configuration settings can be set in the `conf/alluxio-env.
...
+```bash
 $ ./bin/alluxio fs mount --option aws.accessKeyId=<AWS_ACCESS_KEY_ID> --option aws.secretKey=<AWS_SECRET_KEY_ID>\
   /mnt/s3 s3a://<S3_BUCKET>/<S3_DIRECTORY>
 ```
 
 ### Enabling Server Side Encryption
 
-If you are using s3a, you may encrypt your data stored in S3. The encryption is only valid for data
-at rest in s3 and will be transferred in decrypted form when read by clients.
+You may encrypt your data stored in S3. The encryption is only valid for data at rest in s3 and will
```
Comment (aaudiber) [inline docs/en/Configuring-Alluxio-with-S3.md]: nit: we should be consistent about whether S3 is capitalized or not

Comment (calvinjia) [reply docs/en/Configuring-Alluxio-with-S3.md]: Updated, thanks.

**Pull request merged**